<launch>
    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
        <rosparam file="$(find final_assignment)/config/move_base/move_base_params.yaml"
            command="load" />
        <rosparam file="$(find final_assignment)/config/move_base/costmap_common_params.yaml"
            command="load" ns="local_costmap" />
        <rosparam file="$(find final_assignment)/config/move_base/costmap_common_params.yaml"
            command="load" ns="global_costmap" />
        <rosparam file="$(find final_assignment)/config/move_base/local_costmap_params.yaml"
            command="load" />
        <rosparam file="$(find final_assignment)/config/move_base/global_costmap_params.yaml"
            command="load" />
        <rosparam file="$(find final_assignment)/config/move_base/local_planner_params.yaml"
            command="load" />
        <rosparam file="$(find final_assignment)/config/move_base/global_planner_params.yaml"
            command="load" />
        <remap from="/cmd_vel" to="/mobile_base/commands/velocity" />
    </node>

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find final_assignment)/rviz/navigation.rviz"
        required="true" output="screen" />
</launch>